boards/arm/lpc17xx_40xx/u-blox-c027/configs/nsh/defconfig

```diff
@@ -51,8 +50,6 @@ CONFIG_LPC17_40_UART2=y
 CONFIG_LPC17_40_UART3=y
 CONFIG_MM_REGIONS=2
 CONFIG_MODEM=y
-CONFIG_MODEM_U_BLOX=y
```

Comment:
should we enable BSD? since it isn't needed anymore without ublox related code.

Reply:
I think that we should have Apache only configs for all boards (except sim) to avoid the case when a developer accidentally uses mixed licenses. This way each developer will enable the extra licenses locally and benefit from the extra code.

Reply:
this is the nsh config so modem config should not really be there

Reply:
Ok.